certificatemanager/CHANGES.md

```diff
@@ -1,6 +1,19 @@
 # Changes
 
 
+## [0.1.0](https://github.com/googleapis/google-cloud-go/compare/certificatemanager/v0.1.0...certificatemanager/v0.1.0) (2022-04-12)
```

Comment:
This file (certificatemanager/CHANGES.md) now has three entries for `0.1.0` in addition to the original entry for `v0.1.0` that was present when the file was originally generated. Any idea what's wrong?